admin/bin/comfy-admin.js

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+
+const HttpServer = require('http-server');
+const path = require('path');
+
+const server = HttpServer.createServer({ root: path.resolve(__dirname, '../build') });
+server.listen(3000, (error) => {
```

Comment:
We could detect it like create-react-app does and ask the user if we should use another port.
+ We should add a `port` option